crates/client/src/api.rs

```diff
@@ -0,0 +1,22 @@
+use anyhow::Result;
+use reqwest::Url;
+use serde::Deserialize;
+use serde_json::Value;
+
+#[derive(Deserialize, Debug)]
+pub struct Stars {
+    pub amount: Option<u64>,
+}
+
+impl Stars {
+    pub async fn get_amount() -> Result<Self> {
+        let url = format!("https://api.github.com/repos/rustacean-sh/rustacean.sh");
+        let url = Url::parse(&*url)?;
+        let res = reqwest::get(url).await?.text().await?;
+        let v: Value = serde_json::from_str(res.as_str()).unwrap();
```

Comment:
Also recommend using the `Stars` struct directly and defer deserialization to serde here.

```suggestion
        let v: Stars = serde_json::from_str(res.as_str()).unwrap();
```